Server/src/controllers/fileUploadController.ts

```diff
@@ -25,12 +25,12 @@ export class fileUploadController {
     }
     else {
       try {
-        this.callFileUploadService(this.filePathOfUpload, response);
+        this.callFileUploadService(this.filePathOfUpload, response).catch(e => console.log("Failed to run callFileUploadService", e));
```

Comment:
Why was this flagged? It is wrapped in a try catch

Reply:
Because it is an async method it will not be caught properly. Sonarqube suggested to add the .catch()

Reply:
nice